### PR TITLE
Document the centralized build and release team

### PR DESCRIPTION
**Pull request by peternied**
### Description
OpenSearch has been operating with a centralize team for many build and release processes.  This is an attempt to document what this organization model owns to help create understanding of which repository to file requests in to get things done.

### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ORGANIZATION.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# OpenSearch Build / Release Ownership
+
+On OpenSearch there are a number of processes that could be handled by individual development teams - and we are fortunate that there are a number of dedicate team members that can centralize the effort to help these teams scale so smaller teams can keep focused on their areas.  This team is called the OpenSearch DevOps team, it is not exclusive, but it’s dedicated.
+
+This document’s purpose is to clarify where the boundaries between these projects exist and how to get things done.
+
+## Build
+
+OpenSearch is build on code and that code needs to build.  No specific build system is required; however, gradle and yarn are recommendations and will have the most support.
```
Comment (zelinh) [inline ORGANIZATION.md:9]: The first sentence seems confusing. Shall we say "OpenSearch is built on code and that code needs to be built" to be more precise?

File: README.md [hunk 2]
```diff
@@ -15,11 +15,17 @@
 
 This repository contains the scripts for building OpenSearch and OpenSearch Dashboards distributions.
 
-* [OpenSearch Bundle Workflow](bundle-workflow/README.md)
+- [OpenSearch Bundle Workflow](bundle-workflow/README.md)
+
+### Organization
+
+OpenSearch's centralized DevOps team manages centralized build and release processes.
```
Comment (dblock) [inline README.md:22]: Use "centralized" once, e.g. "OpenSearch DevOps team manages a centralized build and release process."

File: ORGANIZATION.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# OpenSearch Build / Release Ownership
```
Comment (dblock) [inline ORGANIZATION.md:1]: I would call this "OpenSearch Project DevOps"

File: ORGANIZATION.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+On OpenSearch there are a number of processes that could be handled by individual development teams - and we are fortunate that there are a number of dedicate team members that can centralize the effort to help these teams scale so smaller teams can keep focused on their areas.  This team is called the OpenSearch DevOps team, it is not exclusive, but it’s dedicated.
+
+This document’s purpose is to clarify where the boundaries between these projects exist and how to get things done.
```
Comment (dblock) [inline ORGANIZATION.md:5]: This document clarifies ...

File: ORGANIZATION.md (hunk 1 above)
Comment (dblock) [inline ORGANIZATION.md:9]: Build -> built

File: ORGANIZATION.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (peternied) [reply ORGANIZATION.md:1]: Today we are build/test heavy, but as we continue on there will be many more packaging related process where we will be spending more time.

File: ORGANIZATION.md (hunk 1 above)
Comment (peterzhuamazon) [reply ORGANIZATION.md:9]: "We need to build OpenSearch Distributions with the code committed to this repo?"

@zelinh's sentence still sounds confusing to me.

**Comment by peternied** [comment]
> My biggest gripe with this doc is that it talks about _Teams_. This is not a concept in OpenSearch, it's an Amazon concept. It's incompatible with the future of open-source where many of these components might as well be managed by people who don't work at Amazon. I think you mean maintainers, who have responsibilities described in https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md. Which means that some of what's prescribed in this doc belongs in that MAINTAINERS.md.

Great suggestion, I'll resolve some of the outstanding writing based issues and then rehome this discussion in that more appropriate location where MAINTAINERS is the concept not a 'team'

**Comment by peternied** [comment]
I am going to drastically reshape and move this content in the OpenSearch repository.  I'll still be circling back to this feedback, but audience perspective will be updated.

**Pull request closed without merging**